src/test/java/org/prebid/server/floors/PriceFloorFetcherTest.java

```diff
@@ -171,6 +177,74 @@ public void fetchShouldCacheResponseForTimeFromResponseCacheControlHeader() {
         verify(vertx).setTimer(eq(700000L), any());
     }
 
+    @Test
+    public void fetchShouldTakePrecedenceForTestingPropertyToCacheResponse() {
+        // given
+        testingProperties.setMinMaxAgeSec(1L);
+        given(httpClient.get(anyString(), anyLong(), anyLong()))
+                .willReturn(Future.succeededFuture(
+                        HttpClientResponse.of(200, MultiMap.caseInsensitiveMultiMap()
+                                        .add(HttpHeaders.CACHE_CONTROL, "max-age=700"),
+                                jacksonMapper.encodeToString(givenPriceFloorRules()))));
+
+        // when
+        priceFloorFetcher.fetch(givenAccount(identity()));
+
+        // then
+        verify(vertx).setTimer(eq(1L), any());
+        verify(vertx).setTimer(eq(1700000L), any());
+    }
+
+    @Test
+    public void fetchShouldTakePrecedenceForTestingPropertyToCreatePeriodicTimer() {
+        // given
+        testingProperties.setMinPeriodSec(1L);
+        given(httpClient.get(anyString(), anyLong(), anyLong()))
+                .willReturn(Future.succeededFuture(
+                        HttpClientResponse.of(200, MultiMap.caseInsensitiveMultiMap(),
+                                jacksonMapper.encodeToString(givenPriceFloorRules()))));
+
+        // when
+        priceFloorFetcher.fetch(givenAccount(identity()));
+
+        // then
+        verify(vertx).setTimer(eq(1L), any());
+        verify(vertx).setTimer(eq(1500000L), any());
+    }
+
+    @Test
+    public void fetchShouldTakePrecedenceForTestingPropertyToChooseRequestTimeout() {
+        // given
+        testingProperties.setMaxTimeoutMs(1L);
+        given(httpClient.get(anyString(), anyLong(), anyLong()))
+                .willReturn(Future.succeededFuture(
+                        HttpClientResponse.of(200, MultiMap.caseInsensitiveMultiMap(),
+                                jacksonMapper.encodeToString(givenPriceFloorRules()))));
+
+        // when
+        priceFloorFetcher.fetch(givenAccount(identity()));
+
+        // then
+        verify(httpClient).get(anyString(), eq(1L), anyLong());
+    }
+
+    @Test
+    public void fetchShouldTakePrecedenceForMinTimeoutTestingPropertyToChooseRequestTimeout() {
+        // given
+        testingProperties.setMinTimeoutMs(1L);
```

Comment:
We can move these constants to separate variables. Probably a thing for refactoring, not actual implementation.

Reply:
In current PBS approach, we are trying not to create constants in test classes